cloud_functions/function.js

```diff
@@ -0,0 +1,72 @@
+const { project } = require('./config')
+const { SecretManagerServiceClient } = require('@google-cloud/secret-manager')
+const fetch = require('node-fetch')
+
+
+const flagCryptominer = async (message, context) => {
+  const data = JSON.parse(Buffer.from(message.data, 'base64').toString())
+  const token = await getSecret({ project, secretName: 'mixpanel-api' })
+  return fetchMixpanelV2('engage#profile-set', {
+    data: {
+      $token: token,
+      $distinct_id: `google:${data.googleSubjectId}`,
+      $set: {
+        'Is Cryptominer': true
```

Comment:
Could this be `isCryptominer` instead? We can always create a translation in the lexicon if we want an English string in the UI.